src/schema/schema.py

```diff
@@ -128,6 +153,17 @@ def from_langchain(cls, message: BaseMessage) -> "ChatMessage":
                     original=original,
                 )
                 return tool_message
+            case TaskMessage():
+                task_message = cls(
+                    type="task",
```

Comment:
Make TaskMessage have type=Task in the class definition, that way you don't need to pass the type here

Reply:
I don't think this will work for two reasons:

1. ChatMessage `type` is not the same as Langchain BaseMessage `type`. They are distinct fields. AIMessage, HumanMessage and ToolMessage have their `type` set in the respective Langchain classes, but we still have to set ChatMessage's type manually as in these other cases. We could write everywhere:
```cls(
  type=message.type,
  ...
```
However, I am not sure of the value of this change.

2. TaskMessage is based on Langchain's `ChatMessage` which has `type` already set to `chat`. I couldn't base it on Langchain's `BaseMessage`, where I could set `type` arbitrarily, because `TaskMessage` needs the message to pass `AnyMessage` Langchain type checks, which Langchain and Langgraph rely on when working with messages. Using `BaseMessage` as base class would fail `AnyMessage` checks.